cirq/ops/three_qubit_gates_test.py

```diff
@@ -32,6 +33,10 @@ def test_eigen_gates_consistent_protocols(eigen_gate_type):
 def test_consistent_protocols():
     cirq.testing.assert_implements_consistent_protocols(cirq.CSWAP)
 
+    diag_angles = [np.pi * i / 4 for i in range(0, 8)]
```

Comment:
Pick angles more arbitrary than this, such as the first 8 primes.

Reply:
OK, used the first 8 primes.